test/test.js

```diff
@@ -7,7 +7,7 @@ const plugin = require('../lib')
 const fixture = file => readFileSync(path.join(__dirname, 'fixtures', `${file}.html`), 'utf8')
 const expected = file => readFileSync(path.join(__dirname, 'expected', `${file}.html`), 'utf8')
 
-const error = (name, cb) => posthtml([plugin()]).process(fixture(name)).catch(cb)
+// const error = (name, cb) => posthtml([plugin()]).process(fixture(name)).catch(cb)
```

Comment:
?

Reply:
I tweaked it a bit to ensure that the content is rendered.
it is not used now, but may be needed.

Reply:
OK then :)